cmd/cluster/create/runner.go

```diff
@@ -73,19 +73,31 @@ func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) err
 	}
 
 	kindCtx := cluster.NewContext(r.flag.Name)
+
 	cfg := &config.Cluster{}
+	nodes := []config.Node{}
 
 	if r.flag.Version != "" {
 		image := fmt.Sprintf("%s:%s", nodeImage, r.flag.Version)
 
-		// Apply image override to all the Nodes defined in Config
-		cfg.Nodes = []config.Node{
-			{
-				Role:  config.ControlPlaneRole,
+		controlPlane := config.Node{
```

Comment:
kind uses control plane I think because that's what kubeadm uses. So I think best to use the same naming here.